server/server.js

```diff
@@ -177,6 +178,17 @@ app.get('/assets/download/:id', (req, res) => {
           const file_stream = storage.get(id);
 
           file_stream.on('end', () => {
+            webPush.sendNotification({
+              endpoint: meta.endpoint,
+              keys: {
+                p256dh: meta.key,
+                auth: meta.authSecret
+              }
+            },
+            meta.filename)
+            .catch(err => {
+              console.log(err);
```

Comment:
It's a non-important error. I actually meant to use `.catch(err=>0)`, but I put it there for debugging purposes for now.